### PR TITLE
[tensorexpr] Enabled aten::stack in the fuser pass with static shapes

### PR DESCRIPTION
**Pull request by huiguoo**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #74077
* #73801

Differential Revision: [D34808051](https://our.internmc.facebook.com/intern/diff/D34808051)

### CODE REVIEW

**Comment by huiguoo** [comment]
@huiguoo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34808051).

File: test/test_jit_fuser_te.py [hunk 1]
```diff
@@ -737,6 +737,22 @@ def foo(hx, cx):
                 # XXX: TE fuser can handle concats in a fusion group.
                 # FileCheck().check("FusedConcat").check_next("return").run(str(graph))
 
+    def test_stack(self):
+        # "aten::stack fusion is not enabled yet with dynamic shapes"
```
Comment (navahgar) [inline test/test_jit_fuser_te.py:741]: What is the reason for this restriction?

Comment (huiguoo) [reply test/test_jit_fuser_te.py:741]: The shape computation formula for aten::stack in JIT is not added in this PR. Will do it in the next PR. It seems a bit compound to have both changes for static and dynamic shapes in one PR.

**Comment by huiguoo** [comment]
@huiguoo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34808051).

**Review by navahgar — APPROVED** [review]
LGTM

**Comment by huiguoo** [comment]
@huiguoo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34808051).

**Comment by huiguoo** [comment]
@huiguoo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34808051).

**Comment by huiguoo** [comment]
@huiguoo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34808051).

**Comment by huiguoo** [comment]
@huiguoo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34808051).

**Pull request closed without merging**